Reachability.m

```diff
@@ -152,6 +156,8 @@ -(instancetype)initWithReachabilityRef:(SCNetworkReachabilityRef)ref
         self.reachableOnWWAN = YES;
         self.reachabilityRef = ref;
 
+        CFRetain(self.reachabilityRef);
```

Comment:
The analyzer is correct. Consider the case where the ```SCNetworkReachabilityRef``` is created, and the ```initWithReachabilityRef:``` call fails. The ref will leak.